changelogs/fragments/441-cloud-init_fix_removal_in_airgapped_environment.yml

```diff
@@ -0,0 +1,4 @@
+---
+bugfix:
+  - hosted_engine_setup - Fix cloud-init package removal in airgapped environment (https://github.com/oVirt/ovirt-ansible-collection/pull/441)
+  
```

Comment:
The link points to the issue (441), please point it to this pr (442).
A new line is still missing, probably extra spaces at line 4.

Reply:
My bad, got confused with the issue.